### ISSUE TITLE
Visual Glitch Minetrain Coaster (RCT1 model) In large Bends

### ISSUE THREAD

**Issue by TomGarstenveld** [OpenRCT2/OpenRCT2#22620]
### Operating System

Windows 10

### OpenRCT2 build

OpenRCT2, v0.4.13-152-g4cfc173 (4cfc173 on develop) provided by GitHub

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

This bug is a graphical glitch or error

### Describe the issue

When The Minetrain Coaster makes a large bend in on one part of the bend the Train goes invicible for a short time! Happened in Millenium Mines in ScenarioPlay. See Screenshot to get the picture.

### Steps to reproduce

I guess make a park in millenium mines with the minetrain coaster. Let it do a large bend and you will encounter the visual bug!

### Attachments

![Millennium Mines 2024-08-24 00-36-46](https://github.com/user-attachments/assets/58bf92b1-b104-4812-b2da-aff2d485a175)

**Comment by Gymnasiast** [comment]
@spacek531 Possibly a similar bug to the Classic Stand-Up?

**Comment by spacek531** [comment]
The draw order is the same between RCT2 and RCT1 objects, so the RCT2 mine train should do the same thing on this coaster.

**Comment by Gymnasiast** [comment]
Didn’t we change a parameter on the Classic Stand-Up to make it glitch less?

**Comment by spacek531** [comment]
Yes, the draw order property was changed. The draw order could be the issue here, but if it is, we should see the same thing happening on RCT2's mine train too.

**Comment by spacek531** [comment]
@TomGarstenveld can you zip your park and upload it to this issue?

**Comment by TomGarstenveld** [comment]
The problem is I don't have it Saved. So I have to recreate it.

**Comment by TomGarstenveld** [comment]
[Millennium Mines.zip](https://github.com/user-attachments/files/16738024/Millennium.Mines.zip)
